cmd/dep/status_test.go

```diff
@@ -382,6 +382,23 @@ func TestCollectConstraints(t *testing.T) {
 			},
 			wantErr: true,
 		},
+		{
+			name: "collect only applicable constraints",
+			lock: dep.Lock{
+				P: []gps.LockedProject{
+					gps.NewLockedProject(
+						gps.ProjectIdentifier{ProjectRoot: gps.ProjectRoot("github.com/JackyChiu/deptest")},
```

Comment:
Let's name the test project something relevant to the test, which would help us recall and refer to it easily. Maybe use words like `applicable`, `constraints`, or anything that you think would be accurate. 😊

Reply:
Sounds good